File: 1537-maximum-score-after-splitting-a-string/1537-maximum-score-after-splitting-a-string.cpp
class Solution {
public:
    int maxScore(string s) {
        int maxi=-1;

        for(int i=1;i<s.length();i++)
        {
            int zc = count(s.begin(),s.begin()+i,'0');
            int oc = count(s.begin()+i,s.end(),'1');

            if(zc+oc > maxi)
                maxi=(zc+oc);
        }

        return maxi;
    }
};

File: 1537-maximum-score-after-splitting-a-string/1537-maximum-score-after-splitting-a-string.py
class Solution:
    def maxScore(self, s: str) -> int:
        maxi=-1
        for i in range(1,len(s)):
            zc=s[0:i].count('0')
            oc=s[i:].count('1')

            if(zc+oc)>maxi:
                maxi=(zc+oc)
        return maxi